src/MongoDB.Driver.Core/Core/Compression/Snappy/Snappy64NativeMethods.cs

```diff
@@ -90,7 +90,7 @@ public override string GetLibraryRelativePath(SupportedPlatform currentPlatform)
                 switch (currentPlatform)
                 {
                     case SupportedPlatform.Windows:
-                        return @"..\..\x64\native\windows\snappy64.dll";
+                        return @"runtimes\win-x64\native\snappy64.dll";
```

Comment:
1. "runtimes" is a magic word. If this word presents (in the `.csproj`), the file `%APP_NAME%.deps.json` in the `bin` folder will additionally contain the following section:
![image](https://user-images.githubusercontent.com/44870738/73767282-50815900-4788-11ea-90ce-1036d91ad353.png)
2. `win-x64` is also magic word. See this link: https://docs.microsoft.com/en-us/dotnet/core/rid-catalog